ios/Classes/TiMapView.m

```diff
@@ -1241,10 +1244,51 @@ - (MKAnnotationView *)mapView:(MKMapView *)mapView viewForAnnotationProxy:(TiMap
     ann.controllerPreviewing = [controller registerForPreviewingWithDelegate:previewingDelegate sourceView:annView];
 #endif
   }
+  float getAngle = [self angleFromCoordinate:userOldLocation toCoordinate:userNewLocation];
+  [UIView animateWithDuration:2
+                   animations:^{
+                     annView.transform = CGAffineTransformMakeRotation(getAngle);
+                   }];
 
   return annView;
 }
 
+- (float)angleFromCoordinate:(CLLocationCoordinate2D)first
+                toCoordinate:(CLLocationCoordinate2D)second
+{
+  float deltaLongitude = second.longitude - first.longitude;
+  float deltaLatitude = second.latitude - first.latitude;
+  float angle = (M_PI * .5f) - atan(deltaLatitude / deltaLongitude);
+
+  if (deltaLongitude > 0)
+    return angle;
+  else if (deltaLongitude < 0)
+    return angle + M_PI;
+  else if (deltaLatitude < 0)
+    return M_PI;
+
+  return 0.0f;
+}
+- (void)updateLoacation:(TiMapAnnotationProxy *)newAnnotation withLocation:(CLLocationCoordinate2D)newLocation
+{
+  MKCoordinateRegion regionInfo;
+  MKCoordinateSpan span;
+  regionInfo.span = span;
+  regionInfo.center = newLocation;
+  [self.map setRegion:regionInfo animated:YES];
```

Comment:
I don't think we should change region. It could cause  a problem if user set multiple location for multiple annotation.

Reply:
This code is used to show user where the notification get lands, as once the annotation got move from screen user will never know where it goes. Do you think we should confirm this from Vijay?